system/jlib/jset.cpp

```diff
@@ -476,6 +480,17 @@ class CBitSet : public CBitSetBase<CBitSetMemoryHelper>
             memset(mem, 0, bitSetUnits*sizeof(bits_t));
         fixedMemory = true;
     }
+    CBitSet(unsigned maxBits, bool reset)
```

Comment:
I thought it would be useful for users to be able to use CBitSet by providing the number of bits required.  It's far simpler than requiring each user to work out how many bytes to allocate or waste a few cycles allocated default size and reallocating.  Surely, as this is a bitset, providing a constructor accepting the number of bits would be a sensible? (The reset parameter has been removed as it should really reset all newly allocated memory )